### PR TITLE
Plotting cleanup

### PR DESCRIPTION
**Pull request by oceanzus**
DO NOT MERGE!!!!!!!!!!!!!!

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@ednad @oceanzus @Bobfrat 

We need to be careful here.  Bob is working on some of this code.   He is going to need a rebase.  Also I think we have a bit of rework with this and Bob's stuff.

**Comment by Bobfrat** [comment]
Looks good to me. Can we merge and I'll rebase my stuff on top of it?

File: ooiservices/app/uframe/controller.py [hunk 1]
```diff
@@ -163,20 +172,24 @@ def streams_list():
     '''
     Accepts stream_name or reference_designator as a URL argument
     '''
-
     if request.args.get('stream_name'):
         try:
             dict_from_stream(request.args.get('stream_name'))
         except Exception as e:
+            print '\n**** (1) exception: ', e.message
```
Comment (birdage) [inline ooiservices/app/uframe/controller.py]: can we remove the print?

File: ooiservices/app/uframe/controller.py [hunk 2]
```diff
@@ -187,36 +200,34 @@ def streams_list():
...
     '''
     try:
-        UFRAME_DATA = current_app.config['UFRAME_URL'] + current_app.config['UFRAME_URL_BASE']
-        current_app.logger.info("GET %s", UFRAME_DATA)
-        timeout = current_app.config['UFRAME_TIMEOUT_CONNECT']
-        timeout_read = current_app.config['UFRAME_TIMEOUT_READ']
-        response = requests.get(UFRAME_DATA, timeout=(timeout, timeout_read))
+        uframe_url, timeout, timeout_read = get_uframe_info()
+        current_app.logger.info("GET %s", uframe_url)
+        print '*** url: ', uframe_url
```
Comment (birdage) [inline ooiservices/app/uframe/controller.py]: rmove print

File: ooiservices/app/uframe/controller.py [hunk 3]
```diff
@@ -187,36 +200,34 @@ def streams_list():
...
     try:
-        UFRAME_DATA = current_app.config['UFRAME_URL'] + current_app.config['UFRAME_URL_BASE'] + '/' + mooring
-        current_app.logger.info("GET %s", UFRAME_DATA)
-        timeout = current_app.config['UFRAME_TIMEOUT_CONNECT']
-        timeout_read = current_app.config['UFRAME_TIMEOUT_READ']
-        response = requests.get(UFRAME_DATA, timeout=(timeout, timeout_read))
+        uframe_url, timeout, timeout_read = get_uframe_info()
+        url = "/".join([uframe_url, mooring])
+        current_app.logger.info("GET %s", url)
+        print '*** url: ', url
```
Comment (birdage) [inline ooiservices/app/uframe/controller.py]: remove print

File: ooiservices/app/uframe/controller.py [hunk 4]
```diff
@@ -248,73 +259,66 @@ def get_uframe_glider_track(ref):
...
     try:
-        UFRAME_DATA = current_app.config['UFRAME_URL'] + current_app.config['UFRAME_URL_BASE'] + '/' + mooring + '/' + platform
-        current_app.logger.info("GET %s", UFRAME_DATA)
-        timeout = current_app.config['UFRAME_TIMEOUT_CONNECT']
-        timeout_read = current_app.config['UFRAME_TIMEOUT_READ']
-        response = requests.get(UFRAME_DATA, timeout=(timeout, timeout_read))
+        uframe_url, timeout, timeout_read = get_uframe_info()
+        url = "/".join([uframe_url, mooring, platform])
+        current_app.logger.info("GET %s", url)
+        print '*** url: ', url
```
Comment (birdage) [inline ooiservices/app/uframe/controller.py]: remove print

**Comment by DanielJMaher** [comment]
@birdage We are going to merge and let bob deal with the prints and stuff

**Comment by DanielJMaher** [comment]
@birdage Unless you want to stop me from merging.  Text, Skype, Flowdock, SMoke Signal... let me know soon.

**Pull request merged**